src/plugin.js

```diff
@@ -57,6 +57,11 @@ const onPlayerReady = (player, options) => {
   });
   // IE11 Windows 8.1+
   player.tech_.el_.addEventListener('msneedkey', (event) => {
+    // EDGE still fires msneedkey, but should use encrypted instead
+    if (videojs.browser.IS_EDGE) {
```

Comment:
Is there a reason that you couldn't put this above the addEventListener, thus avoiding setting the event listener up at all on edge?

Reply:
Nope, nice catch.